STPopup/STPopupController.h

```diff
@@ -174,10 +180,8 @@ typedef NS_ENUM(NSUInteger, STPopupTransitionStyle) {
  */
 - (void)popViewControllerAnimated:(BOOL)animated;
 
-
 /**
  Pops all view controllers from the stack until it reaches the root view controller
-
```

Comment:
Please revert unneeded change if it's possible.

Reply:
ok, done